src/test/java/com/nurkiewicz/java8/J05_MultiRunnerTest.java

```diff
@@ -51,7 +49,7 @@ public void shouldRunMultipleBlocks() throws Exception {
 		));
 
 		//then
-		await().untilCall(to(counter).sum(), is(1L + 2L + 3L));
+		await().until(() -> assertThat(counter.sum()).isEqualTo(1L + 2L + 3L));
```

Comment:
There is AssertJ used already. I see no reason to learn another assertion syntaxt.